.gitignore

```diff
@@ -0,0 +1,93 @@
+
```

Comment:
remove this file please

Reply:
This file is needed in the repository, we can however add it in another MR/

Reply:
We can keep as long as we remove the vendor lines.